pkg/operatorsource/validating.go

```diff
@@ -62,6 +62,6 @@ func (r *validatingReconciler) Reconcile(ctx context.Context, in *marketplace.Op
 
 	r.logger.Info("Scheduling for download")
 
-	nextPhase = phase.GetNext(phase.OperatorSourceDownloading)
+	nextPhase = phase.GetNext(phase.Configuring)
 	return
 }
```

Comment:
There's a comment at the top of this reconciler that mentions the Downloading phase.